letsencrypt-auto-source/tests/auto_test.py

```diff
@@ -349,6 +350,7 @@ def test_openssl_failure(self):
                     self.assertTrue("Couldn't verify signature of downloaded "
                                     "certbot-auto." in exc.output)
                 else:
+                    print(out)
```

Comment:
Did you mean to leave this print statement here?

Reply:
yes.